src/services/meetup/index.ts

```diff
@@ -1,7 +1,7 @@
 import { MEETUP_LINK } from "@/const";
 
 export const getMeetupInfo = async () => {
-  const meetupResponse = await fetch(MEETUP_LINK);
+  const meetupResponse = await fetch(MEETUP_LINK, { cache: "no-store" });
```

Comment:
`{ cache: "no-store" }`
this makes it server side rendered?

Reply:
Correct. This is the app router way of calling getServerSideProps by using options in fetch. [Docs](https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#step-6-migrating-data-fetching-methods)

Reply:
neat 👍